www/js/splash/updatecheck.js

```diff
@@ -45,6 +45,10 @@ angular.module('emission.splash.updatecheck', ['angularLocalStorage'])
          'Ionic Deploy: Update error! '+ res);
     }, function(prog) {
       console.log('Ionic Deploy: Progress... ', prog);
+      $rootScope.progress = prog;
+      $rootScope.isDownloading = true;
+    }).then(function(){
+      $rootScope.isDownloading = false;
```

Comment:
I though this would be promise because update() has a .then() function but this might also be ionic's function. 
The isDownloading is true in the progress callback so I think we do need to set isDownloading to false if there is error during the progress.